openhtf/io/test_record.py

```diff
@@ -29,6 +29,7 @@
 from openhtf.util import logs
 
 _LOG = logging.getLogger(__name__)
+CAPTURE_SOURCE_CODE = True  # Override before creating an openhtf.Test()
```

Comment:
Unfortunately, this happens at the time a phase gets defined, so they'd
need to do it before importing other things.

Otherwise we'd have to change it around to late-load the source code after
the test is created and likely after the args are parsed. Is that worth the
complexity?

On Jul 16, 2016 4:15 PM, "grybmadsci" <notifications@github.com> wrote:

> In openhtf/io/test_record.py
> <https://github.com/google/openhtf/pull/363#discussion_r71073251>:
>
> > @@ -29,6 +29,7 @@
> >  from openhtf.util import logs
> >
> >  _LOG = logging.getLogger(__name__)
> > +CAPTURE_SOURCE_CODE = True  # Override before creating an openhtf.Test()
>
> so what's the intended way for a user to 'Override' this? immediately
> after import via test_record.CAPTURE_SOURCE_CODE = False?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/google/openhtf/pull/363/files/d8a9785b066f7349227d1a8561cf4173695026f3#r71073251>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AASrtLuQn8o4euK2ejXrkQfGE5wWRWMiks5qWWX3gaJpZM4JNtjc>
> .
>